src/styles/App.css

```diff
@@ -60,39 +60,90 @@ body {
   width: 20%;
   height:220px;
   vertical-align:top;
-  margin:2px;
+  margin:3px;
   border-radius: 10px;
   text-align:left;
+  min-width: 250px;
+}
+
+.inner-star {
+  position: absolute;
+  bottom: 0.5rem;
+  right: 0.5rem;
+  z-index: 8999;
+}
+
+.inner-star polygon {
+  fill: #e5ab24 !important;
```

Comment:
Patryk, muszę nadpisywać wartości dla `fill` i `stroke` hardcoded w instancji svg, postaram się to zmienić :)